models/containerized_model/README.md

```diff
@@ -0,0 +1,29 @@
+# How to create containerized models?
+
+## Pre-requisites
+
+- Certifai toolkit (from [CognitiveScale website](https://www.cognitivescale.com/try-certifai/)).
+- A model (pickle) stored in a cloud storage (e.g Amazon S3).
+- A base docker image which (strictly) simulates the trained model python environment.
```

Comment:
I know what you mean by `(strictly) simulates` but we probably need to be more specific and say it has all of the dependencies at the specific versions that were used when the model was trained (or some such)